WcaOnRails/lib/mailer_previews/competitions_mailer_preview.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class MailerPreviews::CompetitionsMailerPreview < ActionMailer::Preview
```

Comment:
Why do you need this class and what does it do?

I'm not sure if it does something relevant for us (and if it does it only does so on 1 competition?).

Reply:
It's a standard way to preview emails in the modern rails version. Because it's built-in. I needed that just for a quick check if I broke something in the mail markup or not.

Reply:
Thanks, I'll resolve this since there is the other thread!